### PR TITLE
refact(charts): make cleanup pre hook image configurable

### PR DESCRIPTION
**Pull request by shubham14bajpai**
Signed-off-by: shubham <shubham.bajpai@mayadata.io>

This PR makes the cleanup pre delete hook job image configurable for air-gap setups

### CODE REVIEW

File: deploy/helm/charts/templates/cleanup-webhook.yaml [hunk 1]
```diff
@@ -26,10 +26,10 @@ spec:
         - name: kubectl
           {{- /* bitnami maintains an image for all k8s versions */}}
           {{- /* see: https://hub.docker.com/r/bitnami/kubectl */}}
-          image: {{ printf "%s/%s:%s.%s" "bitnami" "kubectl" .Capabilities.KubeVersion.Major $kubeMinor }}
+          image: "{{ .Values.cleanup.image.registry }}{{ .Values.cleanup.image.repository }}:{{ .Capabilities.KubeVersion.Major }}.{{ $kubeMinor }}"
```
Comment (prateekpandey14) [inline deploy/helm/charts/templates/cleanup-webhook.yaml:29]: Same comment as openebs/charts PR , tag can be different as well.

File: deploy/helm/charts/values.yaml [hunk 2]
```diff
@@ -238,3 +238,11 @@ analytics:
...
   # Specify in hours the duration after which a ping event needs to be sent.
   pingInterval: "24h"
+
+cleanup:
+  image:
+    # Make sure that registry name end with a '/'.
+    # For example : quay.io/ is a correct value here and quay.io is incorrect
+    registry:
+    repository: bitnami/kubectl
+    tag:
```
Comment (sonasingh46) [inline deploy/helm/charts/values.yaml:248]: Why the tag is empty? Are we going to use `latest` always?

File: deploy/helm/charts/README.md [hunk 3]
```diff
@@ -115,6 +115,8 @@ helm install openebs-cstor openebs-cstor/cstor --namespace openebs --create-name
 | admissionServer.resources | object | `{}` | Admission webhook pod resources |
 | admissionServer.securityContext | object | `{}` | Admission webhook security context |
 | admissionServer.tolerations | list | `[]` | Admission webhook tolerations |
+| cleanup.image.registry | string | `nil` |  cleanup pre hook image registry |
```
Comment (sonasingh46) [inline deploy/helm/charts/README.md:118]: Question(n_p) : 
`nil` vs `""`
( I see that existing code uses `nil` )

File: deploy/helm/charts/values.yaml (hunk 2 above)
Comment (shubham14bajpai) [reply deploy/helm/charts/values.yaml:248]: The tag is determined in the job template itself using the helm Capabilities to determine the k8s version, which will be same as the tag

File: deploy/helm/charts/README.md (hunk 3 above)
Comment (shubham14bajpai) [reply deploy/helm/charts/README.md:118]: The existing values.yaml uses empty fields so I continued with that

**Review by prateekpandey14 — APPROVED** [review]

**Pull request merged**